ENGAGEHF/HealthSummary/HealthSummaryView.swift

```diff
@@ -0,0 +1,82 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import FirebaseAuth
+import FirebaseFunctions
+import PDFKit
+import SpeziViews
+import SwiftUI
+
+
+struct HealthSummaryView: View {
+    @State private var healthSummaryDocument: PDFDocument?
+    @State private var viewState: ViewState = .idle
+    
+    // swiftlint:disable:next accessibility_label_for_image
+    private let sharePreviewImage = Image("HealthSummary")
```

Comment:
Xcode should have a nice way to access the image in a type-safe way without string identifiers from the asset catalogue; we can us this here: https://sarunw.com/posts/swift-symbols-for-asset-catalog/#asset-symbol-generation-in-xcode-15